common/configuration.py

```diff
@@ -44,3 +44,5 @@ def default(cls) -> 'Environment':
 REDIS_DB = config('REDIS_DB', default='0', cast=int)
 
 TOKEN_METADATA_BUCKET = config('TOKEN_METADATA_BUCKET', default=None)
+
+CORS_ALLOWED_ORIGIN = config('CORS_ALLOWED_ORIGIN', default='*')
```

Comment:
Maybe the default should be `null` or `''` (I am not sure what's the opposite of `*`) so we never deploy  with `*` by accident?

Reply:
I think that the default behavior must be always work. It would be very bad if we just make all the service stop working by a simple mistake and i don't think it's too bad to have CORS allowed to everybody for a while.

Reply:
The problem is that if the default is `*` and someone makes a mistake of not setting something different, we will hardly notice the mistake.

And this is a security feature that shouldn't be kept as `*` without notice.

I tend to agree with @pedroferreira1 's suggestion on this one.

Reply:
On the other hand `*` could be susceptible to some attacks, and if we accidentally deploy with it everything will work and we won't notice it until we have to deal with it.

Ideally the deploy should fail if it isn't set to a correct value. And also ideally, it should just work when developing.

Also, IMO we should only have one domain (i.e. not split the explorer into `api.explorer...` and `explorer...`). This would make it so we don't have to set up CORS (which is the safest option, attack-wise).

Reply:
OK, i'm with the team.